### ISSUE TITLE
New release?

### ISSUE THREAD

**Issue by davelab6** [alif-type/mada#7]
Many commits since last release :)

**Comment by davelab6** [comment]
I need TTFs, none are in the repo :(

**Comment by davelab6** [comment]
Here's how I converted the OTF to TTF and hinted it, which is not ideal as it bloats the filesize:

```
$ cat ~/convert.pe 
#!/usr/local/bin/fontforge
i=1
while ( i<$argc )
  Open($argv[i])
  Generate($argv[i]:r + ".ttf")
  i = i+1
endloop
$ ~/convert.pe *otf;
$ for font in *ttf; do \
  ttfautohint -c -I -W -x 0 -D latn -f arab -w G \
  $font \
  $(basename $font .ttf)-ta.ttf; \
done;
$ rename -f s/-ta//g *;
```

**Comment by khaledhosny** [comment]
`make ttf` should generate TTF files, they will be in the release tarball as well.

**Comment by davelab6** [comment]
Sadly I'm not set up for generating

**Comment by khaledhosny** [comment]
I just released 1.0, closing this.
